cocos/ui/UIEditBox/iOS/CCUISingleLineTextField.mm

```diff
@@ -45,4 +41,47 @@ - (CGRect)editingRectForBounds:(CGRect)bounds
     return [self textRectForBounds:bounds];
 }
 
+- (void)setInnerPadding:(float)left :(float)top :(float)right :(float)bottom
+{
+    auto glview = cocos2d::Director::getInstance()->getOpenGLView();
+    float factor = glview->getScaleX() / glview->getContentScaleFactor();
+    
+    _leftPadding = left * factor;
+    _topPadding = top * factor;
+    _rightPadding = right * factor;
+    _bottomPadding = bottom * factor;
+    
+}
+
+- (void)setTextAlignment:(cocos2d::TextHAlignment)hAlign :(cocos2d::TextVAlignment)vAlign
+{
+    switch (hAlign) {
+        case cocos2d::TextHAlignment::LEFT :
+            [self setTextAlignment:NSTextAlignmentLeft];
```

Comment:
`NSTextAlignment` APIs are only available in iOS 6+, but cocos2d-x says it supports iOS 5+.
Personally, I think iOS 5 is just too old, and should be dropped, but that's not my call.